modules/phase_field/src/postprocessors/FeatureFloodCount.C

```diff
@@ -786,7 +924,7 @@ FeatureFloodCount::visitElementalNeighbors(const Elem * elem, unsigned long curr
   std::vector<const Elem *> all_active_neighbors;
 
   // Loop over all neighbors (at the the same level as the current element)
-  for (unsigned int i = 0; i < elem->n_neighbors(); ++i)
+  for (auto end = elem->n_neighbors(), i = 0u; i < end; ++i)
```

Comment:
Yeah, I saw that. I had a similar suggestion a few weeks ago which was not quite as elegant though.